docs/current/assertions.md

```diff
@@ -143,6 +143,26 @@ If only one argument is provided, the assertion passes if `spy` ever threw any e
 
 Like above, only required for all calls to the spy.
 
+#### `sinon.assert.match(actual, expectation);`
+
+Uses [`sinon.match`](./matchers.md) to test if the arguments can be considered a match.
+
+```javascript
+var sinon = require('sinon');
+
+describe('example', function(){
+    it('should fail for mismatching strings', function() {
+        sinon.assert.match('hello', 'world');
```

Comment:
Perhaps it would be worth using an example involving fuzzy matching, since this is what makes `sinon.match` useful compared to say, node or chai assertions?

eg:

```js
// Test value of `x` property, ignore `y`
sinon.assert.match({x:1,y:2}, sinon.match({x:1}))
```